admin-manual/installation-setup/upgrading/upgrading.rst

```diff
@@ -6,47 +6,18 @@ Upgrade from Archivematica |previous_version|.x to |release|
 
 *On this page:*
 
-* :ref:`Clean up completed transfers watched directory <completed-transfers>`
 * :ref:`Create a backup <create-backup>`
 * :ref:`Upgrade Ubuntu package install <upgrade-ubuntu>`
 * :ref:`Upgrade CentOS/Red Hat package install <upgrade-centos>`
 * :ref:`Upgrade in indexless mode <upgrade-indexless>`
 * :ref:`Upgrade with output capturing disabled <upgrade-no-output-capture>`
+* :ref:`Update search indices <update-search-indices>`
 
 .. note::
 
    While it is possible to upgrade a GitHub-based source install using ansible,
    these instructions do not cover that scenario.
 
-.. _completed-transfers:
-
-Clean up completed transfers watched directory
-----------------------------------------------
-
-Upgrading from Archivematica 1.10.x or older to Archivematica 1.11.x can result
-in a number of completed transfers appearing as failed in the Archivematica
-dashboard, as well as corresponding failure notification emails being sent.
-These are not actual failures, but are unintentional side effects of changes
-made in Archivematica 1.11 to the workflow and to how metadata files are stored
-and copied into the SIP.
-
-To prevent these failures from occuring during an upgrade from Archivematica
-1.10 or earlier:
-
-1. Confirm that all transfers and ingests are complete.
-
-   Check that there are no transfers or SIPs that are still being processed or
-   awaiting decisions in the Transfer and Ingest tabs. If there are, finish
-   processing the transfers/ingests before proceeding.
-
-2. Delete all contents of the completedTransfers watched directory.
-
-   .. code:: bash
-
-      sudo rm -rf /var/archivematica/sharedDirectory/watchedDirectories/SIPCreation/completedTransfers/*
-
-3. Perform the upgrade as described below.
-
```

Comment:
I think this is still relevant if you are upgrading from older, isn't it?

Reply:
I was assuming that the instructions assumed users should do a stepped upgrade if jumping between versions, where they upgrade one version at a time. Not sure where I got that - maybe we used to recommend it? Happy to add it back if that's mistaken or we just want to err on the safe side.

Reply:
I can't think of a reason why a stepped upgrade would be needed. @sromkey what do you think?

Reply:
I have no idea! :P

Reply:
Hm, maybe I just made that up. Sounds like we should keep the completed transfers section in then, at least for a few versions!